rust/candid_parser/src/bindings/javascript.rs

```diff
@@ -238,39 +246,87 @@ fn pp_actor<'a>(ty: &'a Type, recs: &'a BTreeSet<&'a str>) -> RcDoc<'a> {
     }
 }
 
+pub fn pp_deprecation_comment<'a>() -> RcDoc<'a> {
```

Comment:
Not really important, but since we use it in the typescript bindings as well, this could be called something like:
```suggestion
pub fn pp_idl_factory_deprecation_comment<'a>() -> RcDoc<'a> {
```

Reply:
Done